ev3dev2/motor.py

```diff
@@ -1920,10 +2002,191 @@ def get_speed_steering(self, steering, speed):
             right_speed *= speed_factor
         else:
             left_speed *= speed_factor
-        
+
         return (left_speed, right_speed)
 
 
+class MoveDifferential(MoveTank):
+    """
+    MoveDifferential is a child of MoveTank that adds the following capabilities:
+
+    - drive in a straight line for a specified distance
+
+    - rotate in place in a circle (clockwise or counter clockwise) for a
+      specified number of degrees
+
+    - drive in an arc (clockwise or counter clockwise) of a specified radius
+      for a specified distance
+
+    New arguments:
+
+    wheel_class - A class typically from ev3dev2/wheel.py. This is used to
```

Comment:
Link to wheel section in docs? There should be examples for motor units. IIRC you might need to manually add a tag to the docs page.

Reply:
Given that none of the other docstrings (none of the ones that I looked at at least...Motor, ColorSensor, etc) are linking to their respective docs sections I would say lets leave it as is.  If we want to have docstrings include links to their docs sections lets do that via a separate issue.

Reply:
The only other case where usage of one class depends on another is with units -- and we have links there. For example:

https://github.com/ev3dev/ev3dev-lang-python/blob/efe9a7398d666f46fda4a6c887bd8e03dfde130f/ev3dev2/motor.py#L890-L895

In these cases the user wants to have quick access to the relevant documentation. Remember, this is used both in docstrings _and_ online rendered docs pages.

Reply:
will fix